features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreaksFragment.kt

```diff
@@ -49,6 +54,16 @@ class FreaksFragment : Fragment() {
         recyclerView.layoutManager = layoutManager
         recyclerView.adapter = ItemAdapter(freaksList)
 
+        val btShowSkills: Button = viewBinding.skillsBtn
```

Comment:
What means btn?

Reply:
It was supposed to show that it's a button. I changed it's name to showSkillsButton for it to be more obvious.